src/overlays/effects/ovl_Effect_Ss_Stick/z_eff_ss_stick.c

```diff
@@ -1,27 +1,56 @@
 /*
  * File: z_eff_ss_stick.c
  * Overlay: ovl_Effect_Ss_Stick
- * Description:
+ * Description: Deku Stick
```

Comment:
This is a fragment, right? So should be "Breaking Deku Stick fragment" or similar, I think.